packages/sanity/src/config/types.ts

```diff
@@ -157,9 +158,31 @@ export interface DocumentPluginOptions {
   badges?: DocumentBadgeComponent[] | DocumentBadgesResolver
   actions?: DocumentActionComponent[] | DocumentActionsResolver
   productionUrl?: AsyncComposableOption<string | undefined, ResolveProductionUrlContext>
+  unstable_languageFilter?: _DocumentLanguageFilterResolver
   newDocumentOptions?: NewDocumentOptionsResolver
 }
 
+/**
+ * @internal
+ */
+export interface _DocumentLanguageFilterContext extends ConfigContext {
```

Comment:
I take the `_`-prefix to mean internal or unstable here - given that we consider the language filter feature to be unstable - would it perhaps add clarity to use the _unstable_ prefix on the accompanying types as well?

Reply:
It uses the `_` prefix since the member is `@internal`. This follows the convention / lint rule defined by API Extractor (Microsoft). Personally think we should follow this convention, but it’s not something we’ve discussed yet.

Reply:
Reference: https://api-extractor.com/pages/messages/ae-internal-missing-underscore/